tests/unit/dataset/dataset_test.py

```diff
@@ -39,6 +39,12 @@
 from pymovements.exceptions import InvalidProperty
 
 
+# pylint: disable=too-many-lines
+
+class _UNSET:
+    ...
+
```

Comment:
why? we could use `from dataclasses import MISSING`

Reply:
Although it would be nice, it is not intended to be used this way:

> No code should directly use the [MISSING](https://docs.python.org/3/library/dataclasses.html#dataclasses.MISSING) value.

https://docs.python.org/3/library/dataclasses.html#dataclasses.field

Relevant stackoverflow post: https://stackoverflow.com/a/69520963